api/java/org/openyolo/api/CredentialClient.java

```diff
@@ -129,8 +144,17 @@ public Intent getHintRetrieveIntent(final HintRetrieveRequest request) {
      * @param callback Handler for the result of the credential request.
      */
     public void retrieve(
-            final CredentialRetrieveRequest request,
+            CredentialRetrieveRequest request,
             final RetrieveCallback callback) {
+
+        // If requireUserMediation was not explicitly set, fallback to isAutoSignInDisabled.
```

Comment:
Logical OR of the two values may be sufficient - I can't think of any cases where you would want to locally override the previously set `autoSignInDisabled` flag stored in the setting. Any use cases in your head?